app/models/manageiq/providers/ibm_cloud/inventory/collector/vpc/target_collection.rb

```diff
@@ -74,7 +76,10 @@ def volumes
   end
 
   def cloud_volume_types
```

Comment:
This should be `volume_profiles` to match https://github.com/ManageIQ/manageiq-providers-ibm_cloud/blob/master/app/models/manageiq/providers/ibm_cloud/inventory/parser/vpc.rb#L311